gcloud-java-storage/src/test/java/com/google/gcloud/storage/testing/RemoteGcsHelperTest.java

```diff
@@ -118,41 +116,44 @@ public String nextPageCursor() {
   @Test
   public void testForceDelete() throws InterruptedException, ExecutionException {
     Storage storageMock = EasyMock.createMock(Storage.class);
+    EasyMock.expect(blob1.blobId()).andReturn(BLOB_ID1);
+    EasyMock.expect(storageMock.delete(BLOB_ID1)).andReturn(true);
+    EasyMock.expect(blob2.blobId()).andReturn(BLOB_ID2);
+    EasyMock.expect(storageMock.delete(BLOB_ID2)).andReturn(true);
     EasyMock.expect(storageMock.list(BUCKET_NAME, BlobListOption.versions(true)))
         .andReturn(blobPage);
-    for (BlobInfo info : blobList) {
-      EasyMock.expect(storageMock.delete(info.blobId())).andReturn(true);
-    }
     EasyMock.expect(storageMock.delete(BUCKET_NAME)).andReturn(true);
-    EasyMock.replay(storageMock);
+    EasyMock.replay(storageMock, blob1, blob2);
     assertTrue(RemoteGcsHelper.forceDelete(storageMock, BUCKET_NAME, 5, TimeUnit.SECONDS));
-    EasyMock.verify(storageMock);
+    EasyMock.verify(storageMock, blob1, blob2);
   }
 
   @Test
   public void testForceDeleteTimeout() throws InterruptedException, ExecutionException {
     Storage storageMock = EasyMock.createMock(Storage.class);
+    EasyMock.expect(blob1.blobId()).andReturn(BLOB_ID1).anyTimes();
+    EasyMock.expect(storageMock.delete(BLOB_ID1)).andReturn(true).anyTimes();
+    EasyMock.expect(blob2.blobId()).andReturn(BLOB_ID2).anyTimes();
+    EasyMock.expect(storageMock.delete(BLOB_ID2)).andReturn(true).anyTimes();
     EasyMock.expect(storageMock.list(BUCKET_NAME, BlobListOption.versions(true)))
         .andReturn(blobPage).anyTimes();
-    for (BlobInfo info : blobList) {
-      EasyMock.expect(storageMock.delete(info.blobId())).andReturn(true).anyTimes();
-    }
     EasyMock.expect(storageMock.delete(BUCKET_NAME)).andThrow(RETRYABLE_EXCEPTION).anyTimes();
-    EasyMock.replay(storageMock);
+    EasyMock.replay(storageMock, blob1, blob2);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [assertTrue(!expr) can be replaced by assertFalse(expr)](https://www.codacy.com/app/mziccard/gcloud-java/file/2386255614/issues/source?bid=3123103&fileBranchId=3228105#l142)